models/eos/ospfv3/eos_ospfv3.yaml

```diff
@@ -0,0 +1,405 @@
+                        description: Specify the SPI value
+                        type: int
+                      algorithm:
+                        description: Specify the params for encryption algorithm.
+                        type: dict
+                        suboptions:
+                          encryption:
+                            description: name of encryption to be used.
+                            type: str
+                            choices: ['3des-cbc', 'aes-128-cbc', 'aes-192-cbc', 'aes-256-cbc', 'null md5', 'null sha1']
+                          algorithm:
+                            description: name of the algorithm to be used.
+                            type: str
+                            choices: ['sha1', 'md5']
+                          encrypt_key:
+                            description: If False, key string is not encrypted
+                            type: bool
+                          hidden_key:
+                            description: If True, Specifies that a HIDDEN key will follow.
+                            type: bool
+                          key:
+                            description: 128 bit MD5 key or 140 bit SHA1 key.
+                            type: str
+                          passphrase:
+                            description: Passphrase String for deriving keys for authentication and encryption.
+                            type: str
+                  nssa:
+                      description: Configures NSSA parameters.
+                      type: dict
+                      suboptions:
+                        default_information_originate:
+                          description: Originate default Type 7 LSA.
+                          type: dict
+                          suboptions:
+                            metric:
+                              description: Metric for default route.
+                              type: int
+                            metric_type:
+                              description: Metric type for default route.
+                              type: int
+                            nssa_only:
+                              description: Limit default advertisement to this NSSA area.
+                              type: bool
+                        no_summary:
+                          description: Filter all type-3 LSAs in the nssa area.
+                          type: bool
+                        nssa_only:
+                          description: Disable Type-7 LSA p-bit setting
+                          type: bool
+                        translate:
+                          description: Enable LSA translation.
+                          type: bool
+                  range:
+                    description: Configure route summarization(not valid when afi=router).
+                    type: dict
+                    suboptions:
+                      address:
+                        description: IP address.
```

Comment:
Should you specify this is IPv6? here and throughout this doc

Reply:
@samccann  Ospfv3 can support both v4 and v6 addresses.